compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/DecodeV1.hs

```diff
@@ -287,6 +290,8 @@ decodeExprSum exprSum = mayDecode "exprSum" exprSum $ \case
       <$> mayDecode "Expr_VariantConTycon" mbTycon decodeTypeConApp
       <*> decodeName VariantConName variant
       <*> mayDecode "Expr_VariantConVariantArg" mbArg decodeExpr
+  LF1.ExprSumEnumCon _ ->
+    Left (ParseError "FixMe")
```

Comment:
better error message

Reply:
done

Reply:
Can you please leave the string "#105" in this error message and all others which refer to enums not being implement in `damlc` yet. I'd like to grep for this string when I implement support for enum types.

Reply:
I forget this one. Thanks for reporting.